GitHubRepositoryMerge.ps1

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+function Merge-GitHubRepositoryBranches
+{
+<#
+    .SYNOPSIS
+        Branch a branch into another
+
+    .DESCRIPTION
+        Merge a branch into another
+        Calls the API: https://developer.github.com/v3/repos/merging/
```

Comment:
I don't think there's a need to explicitly mention this here in the description.  It doesn't add any value to the average user who doesn't know anything about the API, and it's not a practice that is used for any of the other functions in the module.

Reply:
I'll modify that. Thanks for the note